sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -1035,6 +1058,24 @@ public async ValueTask DisposeAsync()
             GC.SuppressFinalize(this);
         }
 
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="maxConcurrentCalls"></param>
+        /// <param name="prefetchCount"></param>
+        public void UpdateConcurrency(int maxConcurrentCalls, int prefetchCount)
```

Comment:
Any reason to not have a separate method UpdatePrefetch?

Reply:
My thought was that concurrency and prefetch should ideally be neatly balanced in relation to each other. That thought made me think you want to tweak those two settings while processing together. 

From an implementation detail perspective having two methods under the assumption both would be mostly used together (if that is true) would require unnecessary double synchronisation

But maybe my assumptions are incorrect when it comes to those two values should be balanced. That being said I always treated it that way in my practice.

What are your thoughts?

Reply:
Happy to adjust to whatever you feel is best

Reply:
Yeah I would just go with a separate method, UpdatePrefetch.